### PR TITLE
Adding ability to manually trigger reschedule - #22

### PR DESCRIPTION
**Pull request by dahiyaAD**

### CODE REVIEW

File: src/main/java/org/icgc/argo/workflow_management_scheduler/controller/RunsController.java
Comment (UmmulkiramR) [inline src/main/java/org/icgc/argo/workflow_management_scheduler/controller/RunsController.java:1]: why do you need to extend the controller from SchedulerStreams?

File: src/main/java/org/icgc/argo/workflow_management_scheduler/rabbitmq/SchedulerStreams.java
Comment (UmmulkiramR) [inline src/main/java/org/icgc/argo/workflow_management_scheduler/rabbitmq/SchedulerStreams.java:1]: consider changing the method name

**Review by UmmulkiramR — CHANGES_REQUESTED** [review]

File: src/main/java/org/icgc/argo/workflow_management_scheduler/controller/RunsController.java
Comment (dahiyaAD) [reply src/main/java/org/icgc/argo/workflow_management_scheduler/controller/RunsController.java:1]: Done!

File: src/main/java/org/icgc/argo/workflow_management_scheduler/rabbitmq/SchedulerStreams.java
Comment (dahiyaAD) [reply src/main/java/org/icgc/argo/workflow_management_scheduler/rabbitmq/SchedulerStreams.java:1]: Done!

**Review by UmmulkiramR — APPROVED** [review]

**Pull request merged**